R/pr.R

```diff
@@ -251,10 +251,9 @@ pr_resume <- function(branch = NULL) {
   check_string(branch)
 
   if (!gert::git_branch_exists(branch, local = TRUE, repo = repo)) {
-    code <- glue('pr_init("{branch}")')
     ui_abort(c(
       "x" = "No branch named {.val {branch}} exists.",
-      "_" = "Call {.code {code}} to create a new PR branch."
+      "_" = "Call {.run usethis::pr_init('{branch}')} to create a new PR branch."
```

Comment:
What's the reasoning for fiddling with this?

Reply:
To create a run link. I tried 2a932519c929d8201c451fce44e584269ba22c25, which seems less invasive?